packages/view/src/components/VerticalClusterList/ClusterGraph/ClusterGraph.hook.tsx

```diff
@@ -71,7 +71,7 @@ const drawClusterGraph = (
 
   group
     .transition()
-    .duration(300)
+    .duration(0)
     .ease(d3.easeLinear)
```

Comment:
duration = 0 이라면 ease 도 없어도 될 것 같은데 검토해주실 수 있나요? 😸 
```suggestion
    .duration(0)
```

Reply:
이거는 혹시 다른분들의 의견이 있을거 같아 남겨놨습니다!! 다시 생각해보니 지우는게 더 깔끔해 보이네요!!